great_expectations/render/renderer/site_builder.py

```diff
@@ -971,6 +977,25 @@ def _add_validations_to_index_links(
                     logger.warning(error_msg)
 
 
+def _resolve_asset_name(validation_results: ExpectationValidationResult) -> str | None:
+    """
+    Resolve the asset name from the validation results meta data.
+    FDS does not store data_asset_name in batch_kwargs or batch_spec and it must be
+    pulled from the active batch definition.
+    """
+    batch_kwargs = validation_results.meta.get("batch_kwargs", {})
+    batch_spec = validation_results.meta.get("batch_spec", {})
```

Comment:
One of both of these may be going away in V1 but `active_batch_definition` will still be present.